src/collection-view.js

```diff
@@ -101,8 +101,9 @@ Marionette.CollectionView = Marionette.View.extend({
   _onCollectionAdd: function(child, collection, opts) {
     var index;
     if (opts.at !== undefined && !this.getOption('filter')) {
-      index = opts.index;
-      if (!index) {
+      if (typeof opts.index === 'number') {
+        index = opts.index;
```

Comment:
This was in gitter, but for documentation's sake.. opts.index is a Bb >= 1.2 feature so this check is essentially `if(Bb < 1.2){`  and the coverage report from travis is run with Bb 1.2.1.  Thus the coverage loss.